packages/build/src/build/buildRefs/buildRefs.test.js

```diff
@@ -396,22 +396,74 @@ describe('vars', () => {
     _ref:
       path: file.yaml
       vars:
-        var2: value`,
+        var1: null`,
       },
       {
         path: 'file.yaml',
         content: `
-  key:
+  key1:
     _var:
       name: var1
+      default: default
+  key2:
+    _var:
+      name: var2
       default: default`,
       },
     ];
     mockReadConfigFile.mockImplementation(readConfigFileMockImplementation(files));
     const res = await buildRefs({ context });
     expect(res).toEqual({
       ref: {
-        key: 'default',
+        key1: null,
```

Comment:
@Gervwyk, @mat02 

I am not sure about this behaviour. If the var is defined, and has value `null`, the default value is not used. I guess this is the behaviour of the `get` function, but this might cause unexpected bugs, since, for example, the `_ref` operator returns `null` if the file is not found.

Reply:
Hmmm, I didn't think it through. Maybe defining var and then setting its value to null is not something that is really needed. Then, `type.isUndefined` should be used as a test function (as you suggested on Discord), not `test.isNone`. Should I change it

Reply:
The issue is that the value of the var is `null`, not the value of the default value. Changing to `type.isUndefined` shouldn't change anything (I believe).

The issue is with the Lowdefy `get` helper. The get function tries to get a value and receives `null`, which is a valid value, and therefore doesn't return the default. All getters in Lowdefy like `_state` will have this behaviour too.

Reply:
I am going to merge this since this case we are discussing is a separate issue with the get helper.